framework/base/Module.php

```diff
@@ -705,4 +705,23 @@ public function afterAction($action, $result)
         $this->trigger(self::EVENT_AFTER_ACTION, $event);
         return $event->result;
     }
+
+    /**
+     * @inheritdoc
+     * Since version 2.0.13, if a component isn't defined in the module, it will be looked up in the parent module.
+     * The parent module may be the application.
+     */
+    public function get($id, $throwException = true)
+    {
+        if (!isset($this->module)) {
+            return parent::get($id, $throwException);
+        }
+        try {
+            return parent::get($id, $throwException);
+        } catch (InvalidConfigException $e) {
+            return $this->module->get($id, $throwException);
+        }
```

Comment:
instead of using try/catch here, I'd better call the parent implementation with `$throwException = false` and test for `null`.